### PR TITLE
Addon-shim was missing dependencies

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
This came up when I moved broccoli-funnel to `nohoist` in a yarn project -- got an error about broccoli-funnel not being found

### CODE REVIEW

**Review by lifeart — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -5756,7 +5756,26 @@ browserslist-generator@^1.0.66:
...
+
+browserslist@^3.2.6:
+  version "3.2.8"
+  resolved "https://registry.yarnpkg.com/browserslist/-/browserslist-3.2.8.tgz#b0005361d6471f0f5952797a76fc985f1f978fc6"
+  integrity sha512-WHVocJYavUwVgVViC0ORikPHQquXwVh939TaelZ4WDqpWgTX/FsGhl/+P4qBUAGcRvtOgDgC+xftNWWp2RUTAQ==
+  dependencies:
+    caniuse-lite "^1.0.30000844"
+    electron-to-chromium "^1.3.47"
+
+browserslist@^4.0.0, browserslist@^4.14.5, browserslist@^4.20.4, browserslist@^4.21.3, browserslist@^4.21.4:
```
Comment (lifeart) [inline yarn.lock:5778]: looks like `yarn-deduplicate` needed here for 3 browserlists

**Comment by ef4** [comment]
Does broccoli-node-api break if you don't change it? Because it's types-only and I don't think our own types refer to them, so it seems like it could stay a devDependency.

As for the other one, we should also figure out why we don't have a lint rule to catch this.

File: yarn.lock (hunk 1 above)
Comment (NullVoxPopuli) [reply yarn.lock:5778]: thanks!

**Comment by NullVoxPopuli** [comment]
yeah, eslint-plugin-node barely supports anything these days (except old cjs)

**Pull request merged**